internal/logging/zerolog.go

```diff
@@ -0,0 +1,61 @@
+package logging
+
+import (
+	"fmt"
+
+	"github.com/rs/zerolog"
+)
+
+// consoleDefaultFormatLevel is copied from zerolog/console.go to modify the
+// names and colors used for levels.
+func consoleFormatLevel(i interface{}) string {
```

Comment:
These were not exported, so I couldn't see any other way of fixing this than to copy it and modify it.